core/domain/question_domain.py

```diff
@@ -412,12 +398,11 @@ def update_question_state_data(self, question_state_data):
 class QuestionSummary(python_utils.OBJECT):
     """Domain object for Question Summary."""
     def __init__(
-            self, creator_id, question_id, question_content,
+            self, question_id, question_content,
             question_model_created_on=None, question_model_last_updated=None):
         """Constructs a Question Summary domain object.
 
         Args:
-            creator_id: str. The user ID of the creator of the question.
```

Comment:
Is there any specific reason to remove this data? Considering that we are going to save the creator_id when we are going to commit the change command, so overall there will be a place where we will be having this info, so any reason to remove this from the summary model?

Reply:
That is the reason. The `creator_id` is saved by the suggestion models, right? So, there is no need to save it here. Also, all topic managers and admins can edit all skills. So, this only matters in other users that suggests skills, which is taken care of.

Reply:
For the suggestion workflow, we are going to have data that stores the creator's info, but for the questions which were created through the question_editor page, where are we going to store this? Are we only going to have the suggestion workflow for questions?

Reply:
Questions, directly through the question editor, can only be created by topic managers and admins, and all topic managers and admins have edit access to them. So, there is no need to store that data when questions are directly created through its editor in the topic or skill editors.
We only need to store user data if it is created outside this, which the suggestion models handle.